src/index.js

```diff
@@ -47,6 +86,23 @@ const renderTasks = () => {
   container.appendChild(list);
 };
 
+const refresh = document.getElementById('refresh');
+refresh.addEventListener('click', () => {
+  renderTasks();
+});
+
+const completed = document.getElementById('removeCompleted');
+completed.addEventListener('click', () => {
+  tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  let tempTasks = tasks;
+  for (let i = 0; i < tasks.length; i += 1) {
+    if (tasks[i].completed === true) {
+      tempTasks = tempTasks.filter((task) => task.index !== tasks[i].index);
+    }
+  }
+  renderTasks();
+});
+
 window.onload = () => {
   renderTasks();
 };
```

Comment:
- [ ] Kindly make sure to fix the reloading problem of your page which is happening every time we add a new todo.

   **Hint:**    _Remove the `window.onload` event and call only your `renderTasks` function_